### DISCUSSION TITLE
Projectiles Can Rebound After Hitting

### DISCUSSION THREAD

**Discussion by Daiki9527** [Phobos-developers/Phobos#1034] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

As we know tanks has tough armor that rebound bullets.We may enable that for all projectiles.

### INI code

```ini
[SomeProjectiles]
Rebounds=yes/no;use for those projectiles without Proximity=yes set.

Rebound.AngleRandom=yes/no;Overwrites These Tour Below
Rebound.MaxXYVel=50
Rebound.MaxZVel=50
Rebound.MinXYVel=50
Rebound.MinZVel=50

Rebound.Arm=128
Rebound.Chance=0.5
Rebound.HitWeapon=WeaponType
Rebound.Anim=Animations
```

**Comment by xindun110** [comment]
very good！
